src/utils/events.js

```diff
@@ -42,7 +42,7 @@ export function convertMilitaryToStandardTime(militaryTime) {
  * @returns {String} The number as a string. It may have  a 0 prepended to it, if it was originally single digit.
  */
 
-export const prependZero = (number) => number < 10 ? "0" + number : number;
+export const prepend0 = (number) => number < 10 ? "0" + number : number;
```

Comment:
This is being used in other places. Please ensure that it is updated everywhere where it is used.